sdk/core/core/README.md

```diff
@@ -120,6 +120,9 @@ Log classifications allow your application to select which specific log messages
    }
    ```
 
+If the SDK is built with `AZ_NO_LOGGING` macro defined (or adding option -DBUILD_LOGGING=OFF with cmake), it should reduce the binary size and slightly improve performance.
+Logging has a negligible performance impact if no listener is registered or if you specify few classifications. However, if you'd like to exclude all of the logging code to make your final executable smaller, define the `AZ_NO_LOGGING` symbol when building the SDK.
+
```

Comment:
We should probably add that without logging, Azure support may not be able to understand or resolve runtime device-caused issues.

Reply:
Thank you Cristian! Logs do not get sent to us either way, so even with the logging enabled, we will not be able to see more that with the logging disabled. Unless customer implements logging callback function that saves logs in some way, and sends it to us some way "by hand", and on our end there is someone whom they will be in contact who agrees to read these logs and help to debug. All this involves some steps - such as us guiding the customer how to better save the logs and send them to us, and if that is the case, we should mention to customer that `NO_LOGGING` should be disabled.
You are probably thinking of something like "unique request ID" policy which we (are supposed to) include with every HTTP request, and it could help us to diagnose problems with the request on our side, but this macro does not control that - it is logging, that is being handled on the device (whether it saves it to a file or sends to some logging server), and the purpose is more to log what's happening on the device, than what is happening on our side.